toy_settings/django_back_end/queries.py

```diff
@@ -24,8 +24,16 @@ def events_for_key(self, key: str) -> list[events.Event]:
 
     def current_value(self, key: str) -> str | None:
         """Get the current value of a setting."""
-        return projections.current_settings(self.events_for_key(key)).get(key)
+        setting = projections.current_settings(self.events_for_key(key)).get(key)
+        if setting is None:
+            return None
+        else:
+            return setting.value
```

Comment:
Thought: I think this strengthens the suggestion of having a null-object.

Reply:
Oops, I think this should just be `...[key].value` -- I made it a default dict so there's always a null object there